misk-redis/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -750,4 +751,55 @@ abstract class AbstractRedisTest {
     assertNull(redis.hget("foo", "bar"))
     assertNull(redis["foo"])
   }
+
+  @Test fun zAddTest() {
+    redis.del("foo");
```

Comment:
Curious if storage is not reset before each test?

Reply:
I don't see anything obvious in the code (like a @BeforeEach). I just tried without those deletes and it still works.